index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css" />
+  <script src="https://kit.fontawesome.com/688ec50e73.js" crossorigin="anonymous"></script>
+</head>
+<body>
+  <header>
+    
+    <h1>velog  </h1>
+    <button><i class="fa-solid fa-magnifying-glass"></i>
+     
+    </button>
+    <button> <i class="fa-solid fa-user"></i>
+    </button>
+  </header>
+  <main>
+    
+      <article>
+        <ul>
+          <li><p id=포스팅1 ></p> <p id=포스팅1></p> <p id=포스팅1></p> <p id=포스팅1></p> <p id=포스팅1></p></li>
+          <li><p id=포스팅1></p> <p id=포스팅1></p> <p id=포스팅1></p> <p id=포스팅1></p> <p id=포스팅1></p></li>
+          <li><p id=포스팅1></p> <p id=포스팅1></p> <p id=포스팅1></p> <p id=포스팅1></p> <p id=포스팅1></p></li>
+       </ul>
```

Comment:
풀리퀘에서 본 어려웠던 점을 보고 말씀드리면, 포스팅 카드 하나하나는 그 구조가 똑같기 때문에 
```suggestion
       </ul>
       <div class="posting-card"> //가장 큰 상자가 되는 컨테이너
           <img src = "이미지 링크"/> //이미지 박스 (img태그는 이미지 밖에 안 올려집니다)
           <div class="text-box"> // 이미지 밑에 텍스트 박스
               <p>타이틀</p>
               <p>설명</p> 
           </div>
       </div>
```

이런 식으로 제일 큰 박스인 posting-card div 태그 사이에 img 태그로 감싸진 img 부분과 text-box div 태그로 감싸진 부분 이렇게 두 가지 영역으로 나뉘고, 다시 text-box div 태그 안에 타이틀과 설명이 들어가는 방식입니다! 이게 바로 우리가 흔히 들은(?) 부모 요소와 자식 요소의 개념입니다. 서로 친구 관계여서 동등한 depth에 있는 건지, 자식이라서 부모의 성질을 계승하게 할 건지 등등을 고려하시면 좋습니당. 
만약에 posting-card의 배경색을 바꾸면 그 아래에 있는 모든 컴포넌트들의 배경색이 바뀌겠지만, text-box의 배경색을 바꾸면 부모 요소인 posting-card나 친구 요소(?)인 img 태그는 영향을 받지 않겠죠?!

이렇게 하나의 posting-card라는 큰 박스를 만들어놓으면, 이제 각기 다른 내용물이 들어가더라도 크게는 posting-card라는 컴포넌트가 반복이 되는 거죠! (물론 배치는 css로 조절)